actors/miner/src/lib.rs

```diff
@@ -216,6 +225,69 @@ impl Actor {
         })
     }
 
+    /// Returns the owner address
+    fn get_owner(rt: &mut impl Runtime) -> Result<GetOwnerReturn, ActorError> {
+        rt.validate_immediate_caller_accept_any()?;
+        let state: State = rt.state()?;
+        let owner = get_miner_info(rt.store(), &state)?.owner;
+        Ok(GetOwnerReturn { owner })
+    }
+
+    /// Returns the worker address
+    fn get_worker(rt: &mut impl Runtime) -> Result<GetWorkerReturn, ActorError> {
+        rt.validate_immediate_caller_accept_any()?;
+        let state: State = rt.state()?;
+        let worker = get_miner_info(rt.store(), &state)?.worker;
+        Ok(GetWorkerReturn { worker })
+    }
+
+    /// Returns the control addresses
+    fn get_controls(rt: &mut impl Runtime) -> Result<GetControlsReturn, ActorError> {
+        rt.validate_immediate_caller_accept_any()?;
+        let state: State = rt.state()?;
+        let controls = get_miner_info(rt.store(), &state)?.control_addresses;
+        Ok(GetControlsReturn { controls })
+    }
```

Comment:
I opened a discussion in Slack about this. We might want to abstract things a bit more and instead add a predicate `is_control_address` that abstracts over representation of this ACL-type thing.

Reply:
(Response copied from Slack for visibility)

Re: control addresses, I think I'm onboard with adding an IsControllingAddress method, with a clear definition of what that means in practice.

I want to flag that I'm not sure how much I like the idea of adding predicates in general. It might put us in an awkward position in the future when the underlying model changes. What if the model changes in such a way that some callers of this predicate would want the new answer to be "no", while others would want to treat it as a "yes"?